File: Favoriti Ispit/eprodajamobile/lib/model/favoriti.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'favoriti.g.dart';

@JsonSerializable()
class Favoriti {
  int? favoritiId;
  int? proizvodId;
  int? korisnikId;
  String? username;
  String? email;

  Favoriti() {}

  factory Favoriti.fromJson(Map<String, dynamic> json) =>
      _$FavoritiFromJson(json);

  /// Connect the generated [_FavoritiToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$FavoritiToJson(this);
}

File: Favoriti Ispit/eProdaja.Model/Favoriti.cs
using System;
using System.Collections.Generic;

namespace eProdaja.Model
{
    public partial class Favoriti
    {
        public int FavoritiId { get; set; }
        public int ProizvodId { get; set; }
        public int KorisnikId { get; set; }
        public DateTime Datum { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
    }
}
